### PR TITLE
Align magnifying glass

### PR DESCRIPTION
**Pull request by dhogan8**
Screenshot 2024-10-01 at 1 35 16 PM

### CODE REVIEW

**Comment by dhogan8** [comment]
https://github.com/metacpan/metacpan-web/issues/3152

File: root/static/less/search.less [hunk 1]
```diff
@@ -23,17 +23,21 @@
     }
 }
 
+.seach-group {
```
Comment (oalders) [inline root/static/less/search.less:26]: Is this a typo?

**Review by oalders — APPROVED** [review]
🔍

**Comment by oalders** [comment]
Thanks, @dhogan8!

**Pull request merged**